packages/plugin-feature/src/feature.js

```diff
@@ -64,6 +64,21 @@ const Feature = SparkPlugin.extend({
       .then((res) => this.spark.device.features[keyType].add(res.body, {merge: true}));
   },
 
+  /**
+   * Issues request to server to set a value for a feature toggle.
+   * @param {array} featureList
+   * @returns {Promise} Refreshes the local device and resolves with the features endpoint`s response.
+   */
+  setBatchUserFeatures(featureList) {
```

Comment:
long term, i think we're going to want to implement this with a batcher anyway, so under-the-hood, it'll probably call `setFeature(type, name, value)`.